landing-page/content/services/data-compaction.html

```diff
@@ -1,7 +1,8 @@
 ---
-Title: Time Travel
-AsciinemaCast: time_travel.cast
-Category: Post
+Title: Data Compaction
+Description: Data compaction is supported out-of-the-box and you can choose from different rewrite strategies such as bin-packing, sorting, or z-ordering to optimize file layout and size.
+LottieFile: data-compaction-animation.json
```

Comment:
I like having an animation, but I don't think that this represents data compaction well. It looks like files are being split instead.

What about having another command box that shows the compaction command, `CALL system.rewrite_data_files(...)`? Then we can link directly to the compaction docs (in 0.13.0).

Reply:
Updated this to show:
```scala
CALL system.rewrite_data_files("nyc.taxis");
```

relevant commit: [b033a57](https://github.com/apache/iceberg-docs/pull/18/commits/b033a571fdb082c317a5987ed1ec5705d3bdce6c)

Reply:
Might want to add a couple options to demonstrate, like `where` to show how to compact a single partition and `strategy` to show bin packing.